torch/onnx/symbolic_opset12.py

```diff
@@ -39,6 +39,9 @@ def nll_loss(g, self, target, weight, reduction, ignore_index):
     # in onnx NegativeLogLikelihoodLoss specification, ignore_index is optional without default value.
     # therefore we need to set ignore_index attribute even if it is not specified (e.g. ignore_index=-100).
     ignore_index = sym_help._maybe_get_const(ignore_index, 'i')
+    if not isinstance(ignore_index, int):
+        return sym_help._onnx_unsupported('ONNX NegativeLogLikelihoodLoss ignore_index need to be known in export time')
```

Comment:
Could ignore_index be None? If that's the case should we just ignore `ignore_index`? `ignore_index` is an optional attribute in ONNX. The above comment is confusing and needs updating I think.